pages/api/retrieval.ts

```diff
@@ -29,14 +30,13 @@ import checkIfValidTemporaryToken from "@lib/middleware/httpRequestHasValidTempT
 async function getFormResponses(
   req: NextApiRequest,
   res: NextApiResponse,
-  formID: string,
-  email?: string,
-  bearerToken?: string
+  { email, temporaryToken }: MiddlewareProps
 ): Promise<void> {
   //Default value to 10 if it's undefined
-  const { maxRecords = "10" } = req.query;
+  const { maxRecords = "10", formID } = req.query;
   //Check maxRecords aren't repeated
-  if (Array.isArray(maxRecords) || !formID) return res.status(400).json({ error: "Bad Request" });
+  if (Array.isArray(maxRecords) || Array.isArray(formID) || !formID)
```

Comment:
If it's going to wrap around anyways might as well add brackets to avoid confusion. Good catch that formID can also be an array

Reply:
:+1